.github/workflows/ci.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```

Comment:
These are branches on external repos, so are not under our control to change.

Reply:
```suggestion
      - uses: actions/checkout@master
```